src/FleetFlow.Api/Middlewares/ExceptionHandlerMiddleware.cs

```diff
@@ -29,6 +31,7 @@ public async Task Invoke(HttpContext context)
             }
             catch(Exception exception)
             {
+                this.logger.LogError(exception.ToString());
```

Comment:
while you are writing to file you need to take some space in order to divide to exeptions. you can paste this code below
 this.logger.LogError($"{exception.ToString()}\n");

Reply:
I corrected it

Reply:
> I corrected it
Nice